src/database/models/mod_item.rs

```diff
@@ -405,7 +405,7 @@ impl Mod {
     }
 
     pub async fn get_full_from_slug<'a, 'b, E>(
-        slug: String,
+        slug: &String,
```

Comment:
Why use a String borrow?
In rust, variable can't be copied without any information, unless they derive the Copy trait.
In our case, string does not derive Copy, but it derives Clone, a verbose copy, meaning that the item is copied only when you call a function, `x.clone()`. In all cases where the string is needed, but you only have a borrow, clone it.

Reply:
Oh.
I actually wanted to replace the String move with a `&str` borrow, but why clone the String?

Reply:
The &str borrow is quite difficult to do, mainly with the lifetime management.
Afterall, it's just a preference, and by convention, only &str and String are allowed

Reply:
I replaced the &String borrow with a &str borrow here, as lifetimes really don't matter here (the slug just goes to the sqlx query and that's it)

Reply:
Perfect then